### ISSUE TITLE
CAD sources are missing

### ISSUE THREAD

**Issue by Goury** [marhod/Orbit#1]
STL is not the source and pretending it is is considered cheating.

**Comment by marhod** [comment]
Agreed.  I didn't say it was...  I just use Github as a repository.   I'm
happy to make my very messy F360 timelines available if that is what you
are after?

**Comment by Clint75** [comment]
you could make step files avalible they are easy to modify if we are allowed to.

**Comment by marhod** [comment]
Done :)
